source/frontend/src/features/mapSideBar/acquisition/add/models.ts

```diff
@@ -21,23 +21,23 @@ export class AcquisitionForm implements WithAcquisitionTeam, WithAcquisitionOwne
   id?: number;
   fileName?: string = '';
   legacyFileNumber?: string = '';
-  assignedDate?: string;
-  deliveryDate?: string;
+  assignedDate?: string = '';
+  deliveryDate?: string = '';
   rowVersion?: number;
   // Code Tables
-  acquisitionFileStatusType?: string;
-  acquisitionPhysFileStatusType?: string;
-  acquisitionType?: string;
+  acquisitionFileStatusType?: string = '';
+  acquisitionPhysFileStatusType?: string = '';
+  acquisitionType?: string = '';
   // MOTI region
-  region?: string;
+  region?: string = '';
   properties: PropertyForm[] = [];
   team: AcquisitionTeamFormModel[] = [];
   owners: AcquisitionOwnerFormModel[] = [];
   fileCheckList: ChecklistItemFormModel[] = [];
 
   project?: IAutocompletePrediction;
   product = '';
-  fundingTypeCode?: string;
+  fundingTypeCode?: string = '';
```

Comment:
kinda nit: If the variables should not be nullable then it would be a good idea to make their types only strings.

Reply:
Fair-ish. This is our desire to make things cleanly nullable pushing up against Formik's hard requirement to not allow formik values fields to be null.

I tried to fix this by making the formik-enabled component set an empty string instead of null where possible, but also wanted to minimize the scope of my changes.